composer.json

```diff
@@ -63,6 +63,9 @@
             "drupal/crop": {
                 "Automated crop integration": "https://www.drupal.org/files/issues/2018-12-16/2830768-crop-automated_crop_integration-45.patch"
             },
+            "drupal/gin": {
+                "Select2 elements do not have a correct styles": "https://www.drupal.org/files/issues/2021-11-01/3246977-ui_improvements_select2_elements-1.patch"
```

Comment:
Isn't this a bugfix that should be landing outside of this PR perhaps?

Reply:
I think make sense to attach this patch here because it's the first task where we add the Select2 widget on an administrative page.